groundstation/static/js/App.jsx

```diff
@@ -172,6 +184,10 @@ function App() {
                               <MenuItem onClick={handleClose}>
                                 <a style={{color: 'rgb(40, 50, 76)'}}href="/logout">Logout</a>
                               </MenuItem>
+                              {isAdmin && 
+                              <MenuItem onClick={handleClose}>
+                                <a style={{color: 'rgb(40, 50, 76)'}}href="/adduser">Add User</a>
+                              </MenuItem> }
```

Comment:
```suggestion
                              {isAdmin && 
                              <MenuItem onClick={handleClose}>
                                <a style={{color: 'rgb(40, 50, 76)'}}href="/adduser">Add User</a>
                              </MenuItem> }
                              <MenuItem onClick={handleClose}>
                                <a style={{color: 'rgb(40, 50, 76)'}}href="/logout">Logout</a>
                              </MenuItem>
```
I think it looks better (and more conventional) if the "Logout" button is at the very last of this list